humble/distribution.yaml

```diff
@@ -5105,6 +5105,22 @@ repositories:
       url: https://github.com/SICKAG/sick_safetyscanners_base.git
       version: ros2
     status: developed
+  simple_actions:
+    doc:
+      type: git
+      url: https://github.com/DLu/simple_actions.git
+      version: main
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/DLu/simple_actions-release.git
+      version: 0.2.1-1
+    source:
+      test_pull_requests: true
```

Comment:
@DLu I assume this has been [setup](http://wiki.ros.org/buildfarm/Pull%20request%20testing) already, right?